### PR TITLE
Added calculation for proportion of wet days.

### LINKED ISSUE #918: Proportion of wet days
Similar to `wet_days`, this would be a method that returns the proportion of days during a period with precipitation over a given threshold.

Original code is by @clairbarnes.

### PR DESCRIPTION
**Pull request by raquel-ucl**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #918 
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)
  - [x] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added.
- [x] `bumpversion patch` has been called on this branch
- [x] The relevant author information has been added to `.zenodo.json`

### What kind of change does this PR introduce?

* Added method to calculate proportion of wet days.

### Does this PR introduce a breaking change?

No

### CODE REVIEW

File: xclim/indices/_threshold.py [hunk 1]
```diff
@@ -1768,6 +1769,44 @@ def wetdays(
...
+
+    >>> from xclim.indices import wetday_prop
+    >>> pr = xr.open_dataset(path_to_pr_file).pr
+    >>> wd = wetday_prop(pr, thresh="5 mm/day", freq="QS-DEC")
+    """
+    thresh = convert_units_to(thresh, pr, "hydro")
+
+    wd = compare(pr, ">=", thresh)
+    fwd = wd.resample(time=freq).mean(dim="time").assign_attrs(units="1")
+    return fwd.rename("fwd")
```
Comment (aulemahal) [inline xclim/indices/_threshold.py:1807]:
```suggestion
    return fwd
```
Same comment as in the other PR.

**Review by aulemahal — COMMENTED** [review]
Nice!

Same as for the other PR, in addition to tests, this still needs:
-  [ ] an indicator
    * [ ] french translation of metadata

File: xclim/data/fr.json [hunk 2]
```diff
@@ -168,6 +168,13 @@
     "title": "Jours mouillés",
     "abstract": "Nombre de jours où les précipitations sont au-dessus d'un seuil."
   },
+  "WETDAY_PROP": {
```
Comment (raquel-ucl) [inline xclim/data/fr.json:171]: @aulemahal here's a placeholder for the french metadata so you can add as a commit suggestion.

**Comment by JamieJQuinn** [comment]
@aulemahal There's now tests for both `wetdays` and `wetdays_prop` indices.

File: xclim/data/fr.json (hunk 2 above)
Comment (huard) [reply xclim/data/fr.json:171]: Done

File: xclim/indices/_threshold.py [hunk 3]
```diff
@@ -1772,6 +1773,44 @@ def wetdays(
...
+    with precipitation over 5 mm at the seasonal frequency, ie DJF, MAM, JJA, SON, DJF, etc.:
+
+    >>> from xclim.indices import wetdays_prop
+    >>> pr = xr.open_dataset(path_to_pr_file).pr
+    >>> wd = wetdays_prop(pr, thresh="5 mm/day", freq="QS-DEC")
+    """
+    thresh = convert_units_to(thresh, pr, "hydro")
+
+    wd = compare(pr, ">=", thresh)
+    fwd = wd.resample(time=freq).mean(dim="time").assign_attrs(units="1")
```
Comment (huard) [inline xclim/indices/_threshold.py:1810]:
```suggestion
    fwd = wd.resample(time=freq).mean(dim="time").assign_attrs(units="")
```

**Review by huard — APPROVED** [review]

File: xclim/indicators/atmos/_precip.py [hunk 4]
```diff
@@ -112,6 +113,16 @@ class HrPrecip(Hourly):
     compute=indices.wetdays,
 )
 
+wetdays_prop = Precip(
+    identifier="wetdays_prop",
+    units="days",
```
Comment (huard) [inline xclim/indicators/atmos/_precip.py:118]:
```suggestion
    units="1",
```

**Comment by Zeitsperre** [comment]
@raquel-ucl Please be sure to bump the version before (`bumpversion patch`). We'll merge after that and the history are updated. Thanks again!

**Comment by JamieJQuinn** [comment]
@Zeitsperre done!

**Comment by Zeitsperre** [comment]
Thanks for the contribution, @raquel-ucl and @JamieJQuinn !

@aulemahal I leave the final review and merge to you.

File: xclim/indicators/atmos/_precip.py [hunk 5]
```diff
@@ -112,6 +113,16 @@ class HrPrecip(Hourly):
     compute=indices.wetdays,
 )
 
+wetdays_prop = Precip(
+    identifier="wetdays_prop",
+    units="1",
+    standard_name="proportion_of_days_with_lwe_thickness_of_precipitation_amount_at_or_above_threshold",
```
Comment (aulemahal) [inline xclim/indicators/atmos/_precip.py:119]:
```suggestion
```
We try to only use standard names present in the official table (http://cfconventions.org/Data/cf-standard-names/78/build/cf-standard-name-table.html).

**Review by aulemahal — APPROVED** [review]
All good!
Only a small change : remove the `standard_name` as it is not an official one.

**Pull request merged**